app/forms/vpc-router-route-common.tsx

```diff
@@ -52,6 +55,42 @@ const targetTypes: Record<Exclude<RouteTarget['type'], 'subnet' | 'vpc'>, string
   drop: 'Drop',
 }
 
+const getDestinationValuePlaceholder = (destinationType: RouteDestination['type']) =>
+  ({
+    ip: 'Enter an IP',
+    ip_net: 'Enter an IP network',
+    subnet: 'Select a subnet',
+    vpc: undefined,
+  })[destinationType]
```

Comment:
I think these are marginally nicer as records and you get the same assurance about missing a key.

```ts
const destinationValuePlaceholder: Record<RouteDestination['type'], string | undefined> = {
  ip: 'Enter an IP',
  ip_net: 'Enter an IP network',
  subnet: 'Select a subnet',
  vpc: undefined,
}
```